EventsExpress.Core/Services/UsersService.cs

```diff
@@ -246,13 +248,23 @@ public IEnumerable<UserDTO> GetUsersByCategories(IEnumerable<CategoryDTO> catego
 
             var users = Db.UserRepository.Get("Photo,Role,Categories.Category")
                 .Where(user => user.Categories
-                    .Any(category => categoryIds.Contains(category.Category.Id)))
+                .Any(category => categoryIds.Contains(category.Category.Id)))
```

Comment:
This tabulation was correct - `Any()` related to `Categories`, not to `UserRepository.Get(...)`